portfolio.html

```diff
@@ -230,5 +233,38 @@ <h1 class="skills-header">Skills</h1>
 
 </section>
 
+    <section class="contact-form">
+        <div class="form-text">
+            <h2 class="text-section">I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.</h2>
+            <form action="https://formspree.io/f/myyvzkbz" method="post">
+                <label for="name">
+                    <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required>
+                </label>
+                <label for="email">
+                  <input type="email" name="email" id="email" class="input-cls" required
+                  placeholder="username@mail.com" pattern="[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{1,63}$" />
+                </label>
+                <label for="message">
+                  <textarea class="input-cls" name="message" id="message" cols="48" rows="9" maxlength="500"
+                  placeholder="Enter text here" required></textarea>
+                </label>
+                <button type="submit" class="btn-green">Get in touch</button>
```

Comment:
- [ ] Kindly, try to fix the button when I move the cursor to button the text "Get in Touch" become overlapped just like the image below : 
![ae](https://user-images.githubusercontent.com/93598189/186731687-7213059a-1219-49a4-b2b6-b5a3bd168736.PNG)